.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,89 @@
+name: Bug report
+description: Submit a new bug report.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## This issue tracker is only for technical issues related to Payjoin Dev Kit.
+
```

Comment:
Bitcoin Core's [bug template](https://github.com/bitcoin/bitcoin/blob/master/.github/ISSUE_TEMPLATE/bug.yml) has mentions of where to go for other issues (namely security & general questions/support) here. I think it would be good for us to have those as well, but I wanted to submit the most helpful template first. I think it would also be helpful to add "Good First Issue" template and "Feature Request" templates, all of which can be easily adapted from Bitcoin Core.